.eslintrc.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
     "prefer-const": 2,
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
+    "react/no-unknown-property": ["error", { ignore: ["transform-origin"] }],
```

Comment:
`transform-origin` is a valid JSX attribute but eslint insists it should be `transformOrigin` which is invalid.